cognite/client/data_classes/data_modeling/views.py

```diff
@@ -0,0 +1,400 @@
+from __future__ import annotations
+
+import json
+from abc import ABC
+from dataclasses import asdict, dataclass
+from typing import TYPE_CHECKING, Any, Dict, Literal, Optional, Union, cast
+
+from cognite.client.data_classes._base import (
+    CogniteFilter,
+    CogniteResourceList,
+)
+from cognite.client.data_classes.data_modeling._validation import validate_data_modeling_identifier
+from cognite.client.data_classes.data_modeling.filters import Filter
+from cognite.client.data_classes.data_modeling.shared import (
+    ContainerReference,
+    DataModeling,
+    DirectNodeRelation,
+    DirectRelationReference,
+    PropertyType,
+    ViewReference,
+)
+from cognite.client.utils._text import (
+    convert_all_keys_to_camel_case_recursive,
+    convert_all_keys_to_snake_case,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+
+class ViewCore(DataModeling):
+    def __init__(
+        self,
+        space: str = None,
+        external_id: str = None,
+        description: str = None,
+        name: str = None,
+        filter: Filter | None = None,
+        implements: list[ViewReference] = None,
+        version: str = None,
```

Comment:
These aren't optional
```suggestion
        space: str,
        external_id: str,
        description: str = None,
        name: str = None,
        filter: Filter | None = None,
        implements: list[ViewReference] = None,
        version: str,
```